### ISSUE TITLE
:owl: The dashed line for the principal value doesn't show up very well on the beeswarm chart

### ISSUE THREAD

**Issue by francisbarton** [The-Strategy-Unit/nhp_reskit#148]
I think we need to use a different colour here, also consider alpha and linewidth.

<img width="1440" height="598" alt="Image" src="https://github.com/user-attachments/assets/8526c065-7ef4-4a8c-ac56-1cf3b18218ab" />

**Comment by matt-dray** [comment]
Aha, at one stage the points were coloured by category, but we don't categorise anymore. That means the point colour is free to vary. Changing the point-colour default in {reskit} would also mean no change is required in outputs code, which has some text that describes the 'red dashed line' or something like that (edit: lol, you can literally see this text in the screenshot).

**Comment by Lextuga007** [comment]
Yes I totally agree on colour changes. Red isn't the easiest to read in text and can signify "bad" which isn't really relevant in this chart. We need to use a default colour scheme I guess and I know @matt-dray has started the big task of looking into that, particularly with contrasts. We have some SU branded colours but there are also [NHS branded](https://www.england.nhs.uk/nhsidentity/identity-guidelines/colours/) too - particularly the highlight colours that are not necessarily so obvious "NHS" like the blue is.

**Comment by Lextuga007** [comment]
Ah that's useful history @matt-dray. I'm not sure but the red in the dots does look like ggplot2 default red - but I can't be sure. That makes the red line (which is red) bolder.

**Comment by francisbarton** [comment]
Lateral thinking! Yes you are right that the point colour was previously an aesthetic. If you're happy for reskit to switch the point colour then I will come up with a good one that contrasts with the vertical lines. Thanks Zoe for the ideas.The red colour is currently just the default `red` in R. There's some alpha applied to the points but not the line.

I agree it would be better to have a different colour than the rather aggressive red but that would require downstream changes in outputs and scenario comparisons and possibly output reports.

**Comment by Lextuga007** [comment]
How much work would be required to go to black for the dashed line? That would be preferable as being dotted is visually different to the full bold line. The contrast too between the grey solid line is possibly not great with the red dots (not sure), and that could feasibly be black too.

The move away from a red line is the first priority with a change to the default red dots next.
